git-remote-gosh/Cargo.toml

```diff
@@ -23,15 +19,18 @@ inherits = 'release'
 
 [dev-dependencies]
 env_logger = "*"
-tracing = {version = "0.1", default-features = false}
-tracing-subscriber = {version = "0.3", default-features = false, features = ["env-filter", "fmt"]}
+tracing = { version = "0.1", default-features = false }
+tracing-subscriber = { version = "0.3", default-features = false, features = [
+  "env-filter",
+  "fmt",
+] }
 
 [dependencies]
 async-trait = '^0'
 base64 = '^0'
 base64-serde = '^0'
 bytes = '^1'
-cached = {version = "^0.38.0", features = ["default"]}
+cached = { version = "^0", features = ["default"] }
```

Comment:
Please keep 0.38
They have a history of breaking changes with minor version updates (0.37 vs 0.38 breaking)